_includes/header.html

```diff
@@ -6,12 +6,12 @@
 
     <nav class="site-nav">
       <div class="trigger">
-        {% for my_page in site.pages %}
-          {% if my_page.title %}
+        {%- for my_page in site.pages -%}
```

Comment:
I have no clue what difference this makes but meh /shrug

Reply:
It trims the whitespace surrounding the Liquid code (tracing in the direction of the % sign).
It’s is called [whitespace control](https://shopify.github.io/liquid/basics/whitespace/).